### PR TITLE
RUBY-1784 Document that retryable writes are now on by default


### PR DESCRIPTION
**Pull request by p-mongo**

### CODE REVIEW

File: docs/tutorials/ruby-driver-create-client.txt [hunk 1]
```diff
@@ -722,30 +722,11 @@ Details on Retryable Writes
...
-elections and similar events, but will not retry writes on network errors
-like timeouts.
-
-Note that writes are always retried on transient server status-related
-errors like "not master" and "node is recovering", regardless of what
-``retry_writes`` option is set to.
-
-The following write methods used in day-to-day operations on collections will
-be automatically retried:
+As of driver version 2.9.0, the modern mechanism is used by default, and the
```
Comment (saghm) [inline docs/tutorials/ruby-driver-create-client.txt]: Am I understanding correctly that on server versions below 3.6 will have no retryable writes (either modern or legacy) by default, and legacy retryable writes will only be enabled on those servers if the `retryWrites: false` option is specified? It's not super clear to me from the changes in the documentation whether this is the case or if legacy retryable writes are enabled by default for older servers still, so I think we should explicitly state whichever way it is.

Comment (p-mongo) [reply docs/tutorials/ruby-driver-create-client.txt]: That is correct, what would you suggest as more clear language?

Comment (saghm) [reply docs/tutorials/ruby-driver-create-client.txt]: I'd change the sentence to "As of driver version 2.9.0, the modern mechanism is used by default on servers that support it, and the legacy mechanism is deprecated and disabled by default on all server versions."

Comment (p-mongo) [reply docs/tutorials/ruby-driver-create-client.txt]: Thank you, made this change.

**Pull request merged**